MiniApp/Classes/core/RealMiniApp.swift

```diff
@@ -346,3 +346,19 @@ extension RealMiniApp: MiniAppMessageDelegate {
         completionHandler(.failure(error as Error))
     }
 }
+
+// MARK: - UI
+extension RealMiniApp {
+    func launch(
+        base: UIViewController,
+        appId: String,
+        config: MiniAppSdkConfig? = nil,
+        messageInterface: MiniAppMessageDelegate? = nil,
+        queryParams: String? = nil
+    ) {
+        let miniAppVC = MiniAppViewController(appId: appId, config: config, messageDelegate: messageInterface ?? self, navDelegate: nil, queryParams: queryParams)
```

Comment:
if you don't have the `MiniApp/UI` in your podfile this will not compile

Reply:
I have a couple of ideas about this:

- We could move the `MiniApp.launch` function into a new class within the `ui` folder like `MiniAppUi`.
- OR we could move the `MiniApp` and `RealMiniApp` extensions for `launch` into the `ui` folder (though I don't know whether this will actually work?)

What do you think?

Reply:
For my part, I think both make sense. 
If I think of firebase for example they've split auth ui into a separate module so it's up to the user but I don't know if there's a case for MiniApp at the current state which benefits from the separation. But in any case separation is almost always cleaner so If we'll add more ui related stuff like eg. List display I'd vote for the first case MiniAppUI.
@Climbatize how about you?

Reply:
As of now, we don't have too many UI dependencies from the Host app. And I see it's good to have a separate module just for the UI is a better option. I already see you had moved the files under the UI folder, But as @Climbatize said, we would have a compilation error if we don't have `MiniApp/UI` in podfile.

Reply:
May be moving extensions of launch into submodule would help 🤔

Reply:
I think you can keep what you did by just moving your extensions to the `ui` folder. By the way having a MiniAppUI class does not bother me, both ways are viable.